pombola/core/views.py

```diff
@@ -508,11 +509,113 @@ def position(request, pt_slug, ok_slug=None, o_slug=None):
         context['alphabetical_link_from_query_parameter'] = True
     context['positions'] = positions
 
-    return render_to_response(
-        template,
-        context,
-        context_instance=RequestContext(request)
-    )
+    if request.GET.get('view') == 'csv':
```

Comment:
My Friday Brain very briefly thought this was referring to a Django View! Perhaps this parameter could be called something like `format` to avoid any ambiguity?

Reply:
👍, that's exactly the word my brain wanted to put in there but couldn't find.